.simplecov

```diff
@@ -0,0 +1,79 @@
+# https://github.com/colszowka/simplecov#using-simplecov-for-centralized-config
+# see https://github.com/colszowka/simplecov/blob/master/lib/simplecov/defaults.rb
+# vim: set ft=ruby
+@minimum_coverage = ENV.fetch('COVERAGE_MINIMUM') { 91.6 }.to_f.round(2)
+# rubocop:disable Style/DoubleNegation
+ENV['FULL_BUILD'] ||= ENV['CI']
+@running_ci       = !!(ENV['FULL_BUILD'] =~ /\Atrue\z/i)
+@generate_report  = @running_ci || !!(ENV['COVERAGE'] =~ /\Atrue\z/i)
+# rubocop:enable Style/DoubleNegation
+
+SimpleCov.profiles.define 'rubygems' do
+  coverage_dir 'reports/coverage'
+  load_profile 'test_frameworks'
+
+  add_group 'Controllers', 'app/controllers'
+  add_group 'Models', 'app/models'
+  add_group 'Helpers', 'app/helpers'
+  add_group 'Libraries', 'lib'
+  add_group 'Middlewares', 'app/middleware'
+  add_group 'Jobs', 'app/jobs'
+  add_group 'API', 'app/controllers/api'
+
+  add_group 'Long files' do |src_file|
+    src_file.lines.count > 100
+  end
+  class MaxLinesFilter < SimpleCov::Filter
+    def matches?(source_file)
+      source_file.lines.count < filter_argument
+    end
+  end
+  add_group 'Short files', MaxLinesFilter.new(5)
+
+  # Exclude these paths from analysis
+  add_filter '/config/'
+  add_filter '/db/'
+  add_filter 'tasks'
+end
+
+## START TRACKING COVERAGE
+require 'coverage'
+Coverage.start
```

Comment:
See PR description for why we want to start coverage tracking before we start SimpleCov.  tl;dr All Rake tasks will require this file.  But not all rake tasks run tests.